pkg/apis/kubermatic/v1/datacenter.go

```diff
@@ -541,8 +541,25 @@ type MeteringConfiguration struct {
 	StorageClassName string `json:"storageClassName"`
 	// StorageSize is the size of the storage class. Default value is 100Gi.
 	StorageSize string `json:"storageSize"`
+	// Schedule in Cron format, see https://en.wikipedia.org/wiki/Cron.
+	// +kubebuilder:default:=`0 1 * * 6`
+	Schedule string `json:"schedule,omitempty"`
+	// Interval defines period for metering timestamps
+	// +kubebuilder:default:=week
+	Interval Interval `json:"interval,omitempty"`
+	// IntervalInDays defines period for metering timestamps. Overwrites Interval.
+	IntervalInDays int `json:"intervalInDays,omitempty"`
```

Comment:
From the comments alone I do not understand why we have 2 flags for the interval. Both seem to define the exact same thing ("period for metering timestamps"). Can "interval" not represent multiple days?

Reply:
From what I see its because the metering tool accepts different flags for the time period. 

But maybe this will be too clunky because we could have a config that looks like this:
```
...
interval: week
intervalInDays: 5
...
```
which will be confusing. 

What about just making one Interval string, no enum. And make a comment that valid values are:
- "yesterday"
- "week"
- "month"
- some int which denotes days?
and you can keep the "week" as default, and handle figuring out what is what in code?

Reply:
Do we need a human readable interval or could it be a ISO 8601 duration (for example "P6T2H") or a cron expression?

Reply:
As @lsviben noticed it was done mainly to mirror flags from our metering report tool (https://github.com/kubermatic/metering/blob/master/cmd/kubermatic-metering-report/main.go#L23-L26). I do agree that having those two flags is very confusing and I shouldn't impose metering tool API on our CRD.

ISO 8601 as I see can define very precise duration (up to seconds) and our tool at the moment supports days as its base unit, which I guess if fine for metering reports. Cron expression doesn't fit as well as we just need to select moment in time from which we want to start calculation.

I decided to have the distinction between `Interval` and `IntervalInDays` because the metering tool works a bit differently in both cases. Not getting into the details I think this difference is negligible.

What do you think about removing `Interval` and leaving just `IntervalInDays` (possibly renamed to `Interval` `int` and documented to stress that the unit are days)?

Reply:
```suggestion
	// Interval defines numer of days taken into report calculation
	// +kubebuilder:default:=7
	Interval int `json:"interval,omitempty"`
```

Reply:
A single `Interval` is IMHO certainly much clearer :)

Reply:
It would simplify implementation as well